nemo/collections/tts/modules/fastpitch.py

```diff
@@ -79,7 +82,7 @@ def average_pitch(pitch, durs):
     return pitch_avg
 
 
-class ConvReLUNorm(torch.nn.Module):
+class ConvReLUNorm(torch.nn.Module, adapter_mixins.AdapterModuleMixin):
```

Comment:
I am wondering if it will be cleaner to move all of the adapter layers into a single adapters.py file, even if requires moving or duplicating some of the generic layers (like this and TemporalPredictor) to the submodules. Then the models definitions themselves will have no explicit adapter logic.